[2023-03-29 15:25:48,866] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: data_merge_dag.start_task manual__2023-03-29T15:25:48.046424+00:00 [queued]>
[2023-03-29 15:25:48,877] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: data_merge_dag.start_task manual__2023-03-29T15:25:48.046424+00:00 [queued]>
[2023-03-29 15:25:48,877] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 15:25:48,877] {taskinstance.py:1357} INFO - Starting attempt 1 of 4
[2023-03-29 15:25:48,877] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 15:25:48,892] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): start_task> on 2023-03-29 15:25:48.046424+00:00
[2023-03-29 15:25:48,897] {standard_task_runner.py:52} INFO - Started process 148 to run task
[2023-03-29 15:25:48,898] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'data_merge_dag', 'start_task', 'manual__2023-03-29T15:25:48.046424+00:00', '--job-id', '428', '--raw', '--subdir', 'DAGS_FOLDER/data_merge.py', '--cfg-path', '/tmp/tmpu9v1x1iw', '--error-file', '/tmp/tmpmxqqytmp']
[2023-03-29 15:25:48,899] {standard_task_runner.py:80} INFO - Job 428: Subtask start_task
[2023-03-29 15:25:48,957] {task_command.py:369} INFO - Running <TaskInstance: data_merge_dag.start_task manual__2023-03-29T15:25:48.046424+00:00 [running]> on host 75e36b0b32e7
[2023-03-29 15:25:49,032] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=kartaca
AIRFLOW_CTX_DAG_ID=data_merge_dag
AIRFLOW_CTX_TASK_ID=start_task
AIRFLOW_CTX_EXECUTION_DATE=2023-03-29T15:25:48.046424+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-29T15:25:48.046424+00:00
[2023-03-29 15:25:49,033] {logging_mixin.py:115} INFO - DAG has been started...
[2023-03-29 15:25:49,033] {python.py:173} INFO - Done. Returned value was: DAG has been started...
[2023-03-29 15:25:49,061] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=data_merge_dag, task_id=start_task, execution_date=20230329T152548, start_date=20230329T152548, end_date=20230329T152549
[2023-03-29 15:25:49,112] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-03-29 15:25:49,157] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
